README.md

```diff
@@ -301,7 +301,6 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Sysco](https://careers.sysco.com/jobs/R79428?lang=en-us) | Houston, TX; or Austin, TX | Intern - Software Development Engineer (10 Week during Summer) |
 | [DoorDash](https://boards.greenhouse.io/doordash/jobs/4560561?gh_jid=4560561) | Multiple Locations | Software Engineer Intern (Summer 2023) |
 | [United](https://careers.united.com/us/en/job/WHQ00020911/Intern-Digital-Technology-Summer-2023) | Chicago, IL | Digital Technology Intern | 
-| [Cornerstone](https://cornerstone.csod.com/ux/ats/careersite/2/home/requisition/7869?c=cornerstone&source=LinkedIn) | Dublin, CA | Software Engineering Intern (Front end) | 
```

Comment:
Can you follow the convention for closing a position? 
<img width="1058" alt="image" src="https://user-images.githubusercontent.com/36709118/190938673-0afabbd3-14da-476c-b327-5f7ed61072a9.png">